lms/djangoapps/support/views/contact_us.py

```diff
@@ -19,6 +20,9 @@ class ContactUsView(View):
     """
 
     def get(self, request):  # lint-amnesty, pylint: disable=missing-function-docstring
+        if not settings.ZENDESK_URL:
+            return redirect('contact')
```

Comment:
Please add appropriate docstring/inline comments to explain why this redirection is necessary.